### PR TITLE
create form cart

### PR DESCRIPTION
**Pull request by ariesh2t**
## Related Tickets:
- [#46229](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-shoe-store/issues/46229)

## WHAT this PR does?
create form cart

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by ariesh2t** [comment]
ready

File: resources/views/users/cart.blade.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+@endsection
+
+@section('content')
+    <!--Timeline items start -->
+<div class="timeline-items container">
+    <h2>{{ __('your cart') }}</h2>
+    <table class="table-cart table table-borderless">
+        <thead>
+            <tr>
+                <th width="10%"></th>
```
Comment (manhtd-1083) [inline resources/views/users/cart.blade.php:14]: bỏ width ra file css riêng

File: resources/views/users/cart.blade.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                        {{ __('place order') }}
+                        <i class="fa-solid fa-angles-right"></i>
+                    </a>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+</div>
+<!--Timeline items end -->
+@endsection
```
Comment (manhtd-1083) [inline resources/views/users/cart.blade.php:70]: eof

**Comment by ariesh2t** [comment]
ready

**Pull request merged**